endpoints-support.md

```diff
@@ -3,16 +3,16 @@
 | Item | Meaning |
 | ------------------ | --------------------------------------------- |
 |  :white_check_mark: | Endpoint implemented in the Ruby SDK for this API version :tada: |
+|  :heavy_multiplication_x: | Endpoint considered as 'out-of-scope' for the Ruby SDK |
 |  :heavy_minus_sign: | Endpoint not available for this API Version |
-|  :x: | Endpoint considered as 'out-of-scope' for the Ruby SDK |
 
 <br />
 
 ***Notes***
 
 * If an endpoint is not marked as implemented for a specific API, it can still be used in compatibility mode for the supported API versions.
 * If an example is not working on a supported API version, verify the [HPE OneView REST API Documentation](http://h17007.www1.hpe.com/docs/enterprise/servers/oneview3.1/cic-api/en/api-docs/current/index.html)  for the API version being used, since the expected attributes for that resource might have changed.
-* For DELETE endpoints which remove multiple resources at once using a filter, that functionality is considered as implemented so long as a DELETE by `{id}` is implemented for that resource.
+* For DELETE endpoints which remove multiple resources at once using a filter, that functionality is considered as implemented as long as a DELETE by `{id}` is implemented for that resource.
```

Comment:
`so` and `as` have the exact same meaning here, but since the sentence already has lots of  `as`, I prefer the use of `so` in this place. Not a must, but I don't see this as a particular improvement.

Reply:
Ok